manifests/k/KDE/KDEConnect/21.08.3/KDE.KDEConnect.installer.yaml

```diff
@@ -1,6 +1,4 @@
-# Created with YamlCreate.ps1 v2.0.2 $debug=QUSU.7-2-0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
```

Comment:
You should take a look at this since more 1.1.0 manifests will be created

Reply:
Just pushed a permanent fix :-)